diagnostics-app/src/app/core/config/diagnostics-params.ts

```diff
@@ -0,0 +1,83 @@
+import {
+    DiagnosticsParams,
+    AcPowerStatus,
+    RoutineType,
+    RunAcPowerRoutineRequest,
+    RunBatteryChargeRoutineRequest,
+    RunBatteryDischargeRoutineRequest,
+    RunBluetoothPairingRoutineRequest,
+    RunBluetoothScanningRoutineRequest,
+    RunCpuRoutineRequest,
+    DiskReadRoutineType,
+    RunDiskReadRequest,
+    NvmeSelfTestType,
+    RunNvmeSelfTestRequest,
+    RunNvmeWearLevelRequest,
+    RunPowerButtonRequest,
+    RunSmartctlCheckRequest,
+} from "@common/dpsl"
+
+const defaultAcPowerRoutineRequest: RunAcPowerRoutineRequest = {
+    expected_status: AcPowerStatus.connected,
+    expected_power_type: 'unknown',
```

Comment:
We should omit this entire line. `expected_power_type` is an optional parameter.

Reply:
Done.